cors/cors_test.go

```diff
@@ -0,0 +1,354 @@
+package cors
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/mount-joy/thelist-lambda/handlers/testhelpers"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_keysToString(t *testing.T) {
```

Comment:
This feels very much like we're testing the implementation details of this package, would it not be normal to only directly test the public API (i.e. the exported methods)?

Reply:
Yeah that's fair - This is because I did the smaller methods first and then didn't see the point deleting the tests